### ISSUE TITLE
Setting `:repl false` causes compile errors on file change

### ISSUE THREAD

**Issue by xeqi** [bhauman/lein-figwheel#107]
## Reproduction

```
$ lein new figwheel ex
$ cd ex
$ sed -i 's/-SNAPSHOT//' project.clj   #Use figwheel 0.2.5

Edit project.clj and set `:repl false`

$ lein figwheel
Wait for compile and connect browser
$ echo " " >> src/ex/core.cljs 
$ echo " " >> src/ex/core.cljs 
```

On the second `echo` a compilation error happens.

## Output

`lein figwheel`:
```
Figwheel: Starting server at http://localhost:3449
Focusing on build ids: dev
Compiling "resources/public/js/compiled/ex.js" from ["src" "dev_src"]...
Successfully compiled "resources/public/js/compiled/ex.js" in 14.82 seconds.
Started Figwheel autobuilder
Figwheel: focusing on build-ids (dev)
Compiling "resources/public/js/compiled/ex.js" from ["src" "dev_src"]...
Successfully compiled "resources/public/js/compiled/ex.js" in 0.172 seconds.
notifying browser that file changed:  resources/public/js/compiled/out/goog/deps.js
notifying browser that file changed:  resources/public/js/compiled/out/cljs_deps.js
notifying browser that file changed:  /ex/core.js
notifying browser that file changed:  /ex/core.js
notifying browser that file changed:  /ex/dev.js
Compiling "resources/public/js/compiled/ex.js" from ["src" "dev_src"]...
Successfully compiled "resources/public/js/compiled/ex.js" in 0.21 seconds.
notifying browser that file changed:  /ex/core.js
notifying browser that file changed:  /ex/core.js
Compiling "resources/public/js/compiled/ex.js" from ["src" "dev_src"]...
WARNING: Use of undeclared Var ex.core/enable-console-print! at line 4 src/ex/core.cljs
WARNING: Use of undeclared Var ex.core/println at line 6 src/ex/core.cljs
WARNING: Use of undeclared Var ex.core/atom at line 10 src/ex/core.cljs
Successfully compiled "resources/public/js/compiled/ex.js" in 0.214 seconds.
notifying browser that file changed:  /ex/core.js
notifying browser that file changed:  /ex/core.js
```

Chrome Developer Console:
```
Edits to this text should show up in your developer console.
utils.cljs?rel=1426528683150:30 Figwheel: trying to open cljs reload socket
utils.cljs?rel=1426528683150:30 Figwheel: socket connection established
utils.cljs?rel=1426528683150:30 Figwheel: notified of file changes
core.cljs?rel=1426528681567:57 Edits to this text should show up in your developer console.
core.cljs?rel=1426528681567:57 Edits to this text should show up in your developer console.
utils.cljs?rel=1426528683150:30 Figwheel: loaded these files
utils.cljs?rel=1426528683150:32 ("ex/core.js" "ex/core.js")
utils.cljs?rel=1426528683150:30 Figwheel: notified of file changes
core.cljs?rel=1426528681567:57 Edits to this text should show up in your developer console.
core.cljs?rel=1426528681567:57 Edits to this text should show up in your developer console.
utils.cljs?rel=1426528683150:30 Figwheel: loaded these files
utils.cljs?rel=1426528683150:32 ("ex/core.js" "ex/core.js")
utils.cljs?rel=1426528683150:29 Figwheel: Compile Warning - Use of undeclared Var ex.core/enable-console-print! at line 4 src/ex/core.cljs
utils.cljs?rel=1426528683150:29 Figwheel: Compile Warning - Use of undeclared Var ex.core/println at line 6 src/ex/core.cljs
utils.cljs?rel=1426528683150:29 Figwheel: Compile Warning - Use of undeclared Var ex.core/atom at line 10 src/ex/core.cljs
client.cljs?rel=1426528682539:81 Figwheel: Not loading code with warnings -  /ex/core.js
utils.cljs?rel=1426528683150:30 Figwheel: notified of file changes
core.cljs?rel=1426528703397:4 Uncaught TypeError: Cannot read property 'call' of undefined
core.cljs?rel=1426528703397:4 Uncaught TypeError: Cannot read property 'call' of undefined
utils.cljs?rel=1426528683150:30 Figwheel: loaded these files
utils.cljs?rel=1426528683150:32 ("ex/core.js" "ex/core.js")
```

## Notes:

Without `:repl false` compilation works fine, even after many, many file changes.

**Comment by bhauman** [comment]
Yep, I'm aware of this.  Thanks for posting the issue. I really think this issue is going to be solved with later versions of cljs. This is a compiler problem that I think is solved now.

**Comment by bhauman** [comment]
Yes I just tested this with 3119 and it appeats to be fixed.

**Comment by bhauman** [comment]
Repl :false doesn't work at all. Runs two auto builders. Fixing now.

**Comment by bhauman** [comment]
fixed on master

**Comment by tillda** [comment]
I ran into this and it took me a LOT of time to figure it out. Would it be possible to create a (pre)release so that people can easily use this in their project.clj file? Thanks!

**Comment by bhauman** [comment]
I have just deployed this fix in 0.2.6
